components/scheduler/edit-appointments.md

```diff
@@ -80,8 +81,148 @@ The UI for the scheduler provides the following options for interacting with the
     * An appointment that spans multiple days but is not marked as an all-day appointment shows up in the all-day slot for the days that it spans entirely. The first and last day would render in the day portions to denote the start and end time accurately.
 
 
+## Delete Confirmation Dialog
 
+The delete confirmation dialog triggers before event deletion. You can enable it by setting the `ConfirmDelete` parameter to `true`. You can also customize the dialog default texts.
```

Comment:
This should include "how" to change the texts.

Reply:
Even if it is a link to the localization article and a few words around it.

Reply:
Done